plugins/github-actions/src/components/WorkflowRunsCard/WorkflowRunsCard.tsx

```diff
@@ -49,7 +45,8 @@ import { buildRouteRef } from '../../routes';
 import { getProjectNameFromEntity } from '../getProjectNameFromEntity';
 import { getHostnameFromEntity } from '../getHostnameFromEntity';
 
-import { Alert, Color } from '@material-ui/lab';
+import Alert from '@material-ui/lab/Alert';
+import { Color } from '@material-ui/lab/';
```

Comment:
@awanlin this one is interesting 

@cftad do you know if there's a more specific path (one level deeper) that this could be taken from?

Reply:
> @cftad do you know if there's a more specific path (one level deeper) that this could be taken from?

`/lab/Alert` should do the trick.